src/utils/get-transfer-data.js

```diff
@@ -81,6 +81,61 @@ function getTransferType(data) {
 }
 
 /**
+ * Get fragment from transfers's `data` if possible, otherwise return null
```

Comment:
Could all of these helpers be replaced by a single `getType(transform, type)` helper instead?